AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerServiceImpl.kt

```diff
@@ -55,6 +57,17 @@ internal class TestRunnerServiceImpl internal constructor(
         firebaseProjectId = firebaseProjectId,
         testMatrixStore = testMatrixStore
     )
+    private val testExecutionStore = TestExecutionStore(
+        toolsResultApi = toolsResultApi,
+        googleCloudApi = googleCloudApi,
+        firebaseTestLabApi = firebaseTestLabApi,
+        firebaseProjectId = firebaseProjectId,
+        datastoreApi = datastoreApi,
+        gcsResultPath = gcsResultPath
+    )
+    private val testResultDownloader = TestResultDownloader(
```

Comment:
for androidx.dev, we shouldn't need this ideally we only use streams there (downloading files is needed for the github Action)